packages/@angular/cli/lib/config/schema.json

```diff
@@ -42,6 +42,10 @@
             "default": "dist/",
             "description": "The output directory for build results."
           },
+          "platform": {
+            "type": "string",
```

Comment:
There is no "enum" type. More like:
"platform": {
  "type": "string",
  "enum": ["browser", "server"],
  "description": "The destination platform of app."
}